generators/app/index.js

```diff
@@ -58,6 +58,9 @@ module.exports = fountain.Base.extend({
           'zone.js': '^0.6.12',
           'es6-promise': '^3.0.2',
           'reflect-metadata': '0.1.3'
+        },
+        devDependencies: {
+          'gulp-inline-ng2-template': '^2.0.4'
```

Comment:
Also, should perhaps be in gulp generator to be consistent with angular 1: https://github.com/FountainJS/generator-fountain-gulp/blob/master/generators/app/index.js#L48